ratatool-cli/src/main/scala/com/spotify/ratatool/tool/Ratatool.scala

```diff
@@ -63,9 +62,6 @@ object Ratatool {
                 val table = BigQueryIO.parseTableSpec(o.tableOut)
                 BigQueryIO.writeToTable(data, sampler.schema, table)
               }
-            case "parquet" =>
-              val data = new ParquetSampler(new Path(o.in)).sample(o.n, o.head)
```

Comment:
We should log something out to the user to raise awareness if people are affected by this move